style.css

```diff
@@ -0,0 +1,66 @@
+@import url(/stylesheets/utilities.css);
+@import url("https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Comfortaa:wght@700&family=Rowdies&display=swap");
+
+:root {
+  /* diferent theme  shades */
+  --theme1: #ffefdb;
+  --theme2: #ffe7c7;
+  --theme3: #ffdeb4;
+  --theme4: #ffd6a0;
+  --theme5: #ffcd8d;
+  --theme6: #ffc579;
+  --theme7: #ffbd65;
+  --theme8: #ffb452;
+  --theme9: #ffa32b;
+  --bg-color: black;
+  --card-bg: rgb(191, 196, 175);
+  --txt-card: rgb(20, 20, 20);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+.nav {
+  background-color: var(--bg-color);
+  color: var(--theme6);
+}
+.nav {
+  border-color: var(--bg-color);
+}
+
+img {
+  border-top-left-radius: 1rem;
+  width: 100%;
```

Comment:
I would suggest to add max-width rather than width for responsiveness.  Also class name for img would be preferable for more specificity.

Reply:
I will work on responsiveness later on, this is just a screen